site/docs/pt/guide/filter-queries.md

```diff
@@ -0,0 +1,355 @@
+Ainda é simples realizar esse tipo de filtragem:
+
+```ts
+bot.on("message").filter(
+  async (ctx) => {
+    const user = await ctx.getAuthor();
+    return user.status === "creator" || user.status === "administrator";
+  },
+  (ctx) => {
+    // Trata mensagens de criadores e administradores.
+  },
+);
+```
+
+### Reutilizando a Lógica de Uma Filter Query
+
+Internamente, o `bot.on` depende de uma função chamada `matchFilter`.
+Ela recebe uma filter query e a compila para uma função de predicado.
+O predicado é simplesmente passado para `bot.filter` para filtrar atualizações.
+
+Você pode importar `matchFilter` diretamente se quiser usá-lo em sua própria lógica.
+Por exemplo, você pode decidir descartar todas as atualizações que correspondem a uma determinada query:
+
+```ts
+// Descarta todas as mensagens de texto ou posts de text em canais.
+bot.drop(matchFilter(":text"));
+```
+
+Analogamente, você pode fazer uso dos tipos de filter query que o grammY usa internamente:
+
+### Reutilizando os Tipos de Filter Query
+
+Internamente, `matchFilter` usa os [type predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) do TypeScript para ajustar o tipo de `ctx`.
+Ela recebe um tipo `C extends Context` e um `Q extends FilterQuery` e produz `ctx is Filter<C, Q>`.
+Em outras palavras, o tipo `Filter` é o que você realmente recebe como seu `ctx` no middleware.
+
+Você pode importar `Filter` diretamente se quiser usá-lo em sua própria lógica.
+Por exemplo, você pode decidir definir uma função de handler que trata objetos de contexto específicos que foram filtrados por uma filter query:
+
+```ts
+function handler(ctx: Filter<Context, ":text">) {
+  // trata objeto de contexto ajustado
+}
+
+bot.on(":text", handler);
+```
+
+> Confira as referências da API para [`matchFilter`](https://deno.land/x/grammy/filter.ts?s=matchFilter), [`Filter`](https://deno.land/x/grammy/filter.ts?s=Filter) e [`FilterQuery`](https://deno.land/x/grammy/filter.ts?s=FilterQuery) para ler mais.
+
+## A Linguagem de Query
+
+> Esta seção é destinada a usuários que desejam ter um entendimento mais profundo das filter queries no grammY, mas não contém nenhum conhecimento necessário para criar um bot.
+
+### Estrutura da Query
+
+Toda query consiste em até três pedaços de query.
+Dependendo de quantos pedaços de query uma query tem, diferenciamos entre queries L1, L2 e L3, como `"message"`, `"message:entities"` e `"message:entities:url"`, respectivamente.
+
+Os pedaços de query são separados por dois pontos (`:`).
+Nos referimos ao pedaço anterior aos dois pontos ou ao final da string de query como _pedaço L1_ de uma query.
+Nos referimos ao pedaço dos dois pontos ao segundo dois pontos ou ao final da string de query como _pedaço L2_ da query.
+Nos referimos ao pedaço dos dois pontos ao final da string de query como _pedaço L3_ da query.
+
+Exemplo:
+
+| Filter Query                 | Pedaço L1   | Pedaço L2    | Pedaço L3   |
```

Comment:
```suggestion
| Filter Query                 | Fragmento L1   | Fragmento L2    | Fragmento L3   |
```

I have to add more spaces here to align the columns but in this github review section I can't. I'll try another way of edit that tomorrow, just comenting here to remember.